lib/rules.js

```diff
@@ -224,6 +224,26 @@ const wordPrefixRuleCreator = word =>
     { name: `wordPrefixRule '${word}'` }
   );
 
+const hyphensOmittedOrReplacedWithSpaces = new FunctionRule(
```

Comment:
If we identify any hyphen lookalikes that we want to accept as correct, then you can see how if we added a MapRule before this one that replaces all of them with ASCII `-`, this rule will simply work for them as well as it does for `-`.